addons/sale_project/models/product_template.py

```diff
@@ -49,6 +53,12 @@ def _compute_service_policy(self):
             if not product.service_policy and product.type == 'service':
                 product.service_policy = 'ordered_prepaid'
 
+    @api.depends('project_id')
+    def _compute_task_template(self):
+        for product in self:
+            if product.task_template_id not in product.project_id.task_ids:
```

Comment:
```suggestion
            if product.task_template_id and product.task_template_id.project_id != product.project_id:
```

Why do we no longer take the project_template_id into account? I thought it was the case before your recent changes